packages/kbn-dependency-ownership/src/rule.ts

```diff
@@ -26,11 +27,18 @@ export function ruleFilter(rule: RenovatePackageRule) {
     'typescript', // These updates are always handled manually
     'webpack', // While we are in the middle of a webpack upgrade. TODO: Remove this once we are done.
   ];
+  // Rules that use custom managers are not supported by this tool, and are ignored.
+  const rulesWithCustomManagers = ['chainguard', 'chainguard-fips'];
+
   return (
     // Only include rules that are enabled or explicitly allowed to be disabled
     (allowedDisabledRules.includes(rule.groupName) || rule.enabled !== false) &&
     // Only include rules that have a team reviewer
-    rule.reviewers?.some((reviewer) => reviewer.startsWith('team:'))
+    rule.reviewers?.some((reviewer) => reviewer.startsWith('team:')) &&
+    // Only include rules that use the default manager, or specify npm
+    (!rule.matchManagers || rule.matchManagers.includes('npm')) &&
```

Comment:
Should this also check the matchMangers array length? True if not defined/length is not  > 0

```suggestion
    (!rule.matchManagers?.length > 0 || rule.matchManagers.includes('npm')) &&
```